universal-login-commons/test/core/utils/validateEmail.test.ts

```diff
@@ -0,0 +1,36 @@
+import {isProperEmail} from '../../../src/core/utils/isProperEmail';
+import {expect} from 'chai';
+
+describe('UNIT: ValidateEmail', () => {
+  it('Should validate valid email', () => {
+    expect(isProperEmail('valid@email.com')).be.eq(true);
+  });
+
+  describe('Should not validate invalid email', () => {
```

Comment:
Does this function validate smth or return false? Name this test: 'invalid email'